styles.css

```diff
@@ -0,0 +1,87 @@
+body {
+  margin: 0;
+  height: 100%;
+  width: 100vw;
+  overflow: hidden;
+}
+
+.page-top {
```

Comment:
- [ ] It would be better if we address the spacing/margin of the logo and the hamburger, let us center it and add a little bit of font-weight to it